DESCRIPTION

```diff
@@ -84,13 +84,16 @@ VignetteBuilder:
     knitr,
     rmarkdown
 Remotes:
+    insightsengineering/teal@main,
+    insightsengineering/teal.code@main,
+    insightsengineering/teal.data@main,
+    insightsengineering/teal.logger@main,
```

Comment:
Missing remotes, leaving a comment to block until we discuss if this should be complete